api/SourceBuffer.json

```diff
@@ -1723,7 +1678,14 @@
             "description": "Available in workers",
             "support": {
               "chrome": {
-                "version_added": "108"
+                "version_added": "108",
```

Comment:
Is there a way to note that both SourceBuffer.audioTracks and .videoTracks in a *dedicated worker* MediaSource's SourceBuffer is *not* expected to succeed even if the experimental flag is provided (there is further spec work and implementation work required to support those track lists from a worker context, and currently the implementation - if experimental flag is enabled and the app invokes the audioTracks() or videoTracks() getter on a *worker* SourceBuffer, then Chrome crashes the renderer process to avoid potentially worse behavior. This is known work in both spec and impl (MSE spec, HTML spec, Chrome impl), partially captured by this MSE spec issue: https://github.com/w3c/media-source/issues/280

Reply:
I've added a note to `audioTracks`/`videoTracks` to hopefully cover this; let me know what you think. I wasn't sure that the spec issue you linked to really explained what the issue was, so I decided not to link to it. IS there a better link available, or do you think I should link here, as that is where explanation will be added?

Reply:
Good point - I've reactivated the most closely related Chromium issue to this piece of the spec. Please link to https://bugs.chromium.org/p/chromium/issues/detail?id=487288#c24 (it also links transitively to the MSE spec issue 280 that I mentioned, above).

Reply:
Perfect, that is a useful explanation. Link added.